.github/workflows/release.yml

```diff
@@ -0,0 +1,150 @@
+
+  build-group-publish:
+    runs-on: ubuntu-latest 
+    needs: load-build-group
+    strategy:
+      fail-fast: false
+      matrix: ${{fromJson(needs.load-build-group.outputs.matrix)}}
+    container:
+        image:  ${{ matrix.image }}
+
+    env: 
+      AWS_ACCESS_KEY_ID: ${{ secrets.PROD_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.PROD_AWS_SECRET_ACCESS_KEY }}
+
+    steps:
+      - name: Checkout ${{ github.ref }}
+        uses: actions/checkout@v2
+
+      - name: Show Environment Info
+        run: |
+          printenv
+          node --version
+          npm --version 
+          cat /etc/os-release
+
+      # must install specific dependencies before a build
+      # can't call npm install. doing so may fallback-to-build if package has yet to be published (double build)
+      # use npm workaround specifying a package name to bypass install script in package.json
+      - name: NPM Install dependencies
+        run: npm install linux-os-info --unsafe-perm
+
+      # must setup libobo before build
+      # node-pre-gyp rebuild runs "clean" and "build" at once.
+      # it is mapped to `node-gyp rebuild` which internally means "clean + configure + build" and triggers a full recompile
+      - name: Build
+        run: |
+          node setup-liboboe.js
+          npx node-pre-gyp rebuild
+
+      # artifacts are at:build/stage/nodejs/bindings/
+      - name: Package
+        run: npx node-pre-gyp package # requires clean rebuild
+
+      # *** IMPORTANT: 
+      # always include --s3_host flag regardless of node-pre-gyp defaults.
+      # workflows designed to ensure that the  staging bucket already has similarly versioned package
+      - name: Publish Package to Production
+        run: npx node-pre-gyp publish --s3_host=production
+
+  load-target-group:
+    runs-on: ubuntu-latest
+    needs: build-group-publish
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout ${{ github.ref }}
+      uses: actions/checkout@v2
+
+    - name: Load target group data
+      id: set-matrix
+      run: .github/scripts/matrix-from-json.sh .github/config/target-group.json
+
+  target-group-install:
+    runs-on: ubuntu-latest 
+    needs: load-target-group
+    strategy:
+      fail-fast: false
+      matrix: ${{fromJson(needs.load-target-group.outputs.matrix)}}
+
+    steps:
+      - name: Checkout ${{ github.ref }}
+        uses: actions/checkout@v2
+
+      - name: Show Environment Info
+        run: |
+          printenv
+          node --version
+          npm --version 
+          cat /etc/os-release
+
+      - name: NPM Install Production
+        run: npm install --unsafe-perm --s3_host=production
+
+      - name: Check Artifacts
+        run: ls ./dist/napi-v*/apm_bindings.node && ls ./dist/napi-v*/ao_metrics.node
+
+  npm-publish:
+    runs-on: ubuntu-latest 
+    if: ${{ startsWith(github.ref, 'refs/tags/') }} # will not run when manually triggered
```

Comment:
This won't be needed at the moment, as there is no manual trigger option.

Reply:
Correct. Given how trigger filtering work (or don't) I'm a little (irrationally maybe?) hesitant to remove it though...

Reply:
Left the conditional, changed the comment.